### DISCUSSION TITLE
Arrows prevent blocks from being placed

### DISCUSSION THREAD

**Discussion by cody-quinn** [Minestom/Minestom#982] [Help] [answered]
Arrows are preventing blocks from being placed, is there any way to exclude arrows/specific entities from preventing block placement? Thanks.

Edit: In addition to this players in spectator

**Comment by ArvedEh** [comment, marked as answer]
Hi, this is currently not implemented. This should probably be an issue (?)
I think I'll add the checks and make a PR.

Edit: Merged. This should work now.
